index.html

```diff
@@ -5638,7 +5634,8 @@ <h4 id="grouping_of_characters_and_symbols_depending_on_their_positioning">
           <p id="cl-14-en" its-locale-filter-list="en" lang="en">Full-width ideographic space (cl-14)</p>
           <p id="cl-14-ja" its-locale-filter-list="ja" lang="ja">和字間隔（cl-14）</p>
         <div class="example">
-            <p its-locale-filter-list="en" lang="en">U+3000 IDEOGRAPHIC SPACE</p>
+            <p its-locale-filter-list="en" lang="en">U+3000 (IDEOGRAPHIC SPACE)</p>
+            <p its-locale-filter-list="ja" lang="ja">U+3000</p>
```

Comment:
Why is this going to give different results in English & Japanese?

If you remove the original its-locale-filter-list attribute, i think it should remain the same for both languages.

Reply:
it's just my mistake when I've added ja version. fixed.